### PR TITLE
Teet 1360 html export

### PR DESCRIPTION
**Pull request by jkorpelin**
html export for land owner opinions

### CODE REVIEW

**Review by erno — COMMENTED** [review]
Code looks generally good, but some high level comments about what it's doing would be good for the travellers of future. eg what's jrk-nr, what the general structure of the resulting page is (per-unit tables about opinions / positions, and a summary table, from a cursory glance).

File: app/backend/test/teet/integration/x_road_test.clj [hunk 1]
```diff
@@ -184,8 +184,8 @@
                           property-registry/kinnistu-d-parse-response)
           omandiosad (:omandiosad parsed-resp)]
       (is (= :ok (:status parsed-resp)))
-      
-      
+
+
```
Comment (outamaa) [inline app/backend/test/teet/integration/x_road_test.clj:188]: Is this intentional?

**Review by tatut — APPROVED** [review]

File: app/backend/src/clj/teet/land/owner_opinion_export.clj [hunk 2]
```diff
@@ -14,14 +17,133 @@
...
          db activity type)
-    (db-api-large-text/with-large-text #{:cooperation.response/content :cooperation.opinion/comment})
+    (db-api-large-text/with-large-text #{:land-owner-opinion/authority-position :land-owner-opinion/body})
     (map first)))
 
-(defn summary-table [db activity type]
-  (let [opinions (opinions-by-type db activity type)]
+(def tr* #(tr [:land-owner-opinion :export %]))
+
+(defn jrk-nr
```
Comment (outamaa) [inline app/backend/src/clj/teet/land/owner_opinion_export.clj:25]: As Erno said, functions in this ns could use descriptive comments for future generations.

File: app/backend/src/clj/teet/land/owner_opinion_export.clj [hunk 3]
```diff
@@ -14,14 +17,133 @@
...
+(defn summary-table [db activity opinion-type {:keys [api-url api-secret] :as _config}]
+  (let [opinions (opinions-by-type db activity opinion-type)
+        project (ffirst
+                  (d/q '[:find (pull ?project [:thk.project/name :thk.project/project-name
+                                               :db/id :thk.project/id])
+                         :in $ ?activity
+                         :where
+                         [?lifecycle :thk.lifecycle/activities ?activity]
+                         [?project :thk.project/lifecycles ?lifecycle]]
+                       db activity))
```
Comment (aleksievelizhanin) [inline app/backend/src/clj/teet/land/owner_opinion_export.clj:121]: This code is essentially a function from project namespace and can be refactored as such.

**Review by outamaa — APPROVED** [review]

**Review by aleksievelizhanin — APPROVED** [review]
Looks legit. Can be improved further by minor refactoring.

File: app/backend/test/teet/integration/x_road_test.clj (hunk 1 above)
Comment (jkorpelin) [reply app/backend/test/teet/integration/x_road_test.clj:188]: For some reason Ernos editor leaves these empty spaces the files sometimes and Idea just automatically removes them if I open the file.

**Pull request merged**